addons/medical_statemachine/Statemachine.hpp

```diff
@@ -105,6 +105,10 @@ class ACE_Medical_StateMachine {
             condition = QFUNC(conditionExecutionDeath);
             events[] = {QEGVAR(medical,FatalInjury)};
         };
+        class Bleedout {
```

Comment:
This transition needs to be added to the other states as well. Otherwise if blood volume drops significantly you could potentially by pass the cardiac arrest state and end up with bleedout events without transitioning into a different state.